src/stores/SCWPStore.ts

```diff
@@ -0,0 +1,10 @@
+import { proxy } from 'valtio'
+import PersistableStore from 'stores/persistence/PersistableStore'
+
+class SCWPStore extends PersistableStore {
+  announceClosed = false
+}
+
+const scwpStore = proxy(new SCWPStore()).makePersistent(true)
```

Comment:
I guess, that can avoid `vonst` here, simply `export proxy(...).makePersistent(true)`